runtime/rococo/src/lib.rs

```diff
@@ -235,6 +235,8 @@ construct_runtime! {
 		Mmr: pallet_mmr::{Pallet, Call, Storage},
 		Beefy: pallet_beefy::{Pallet, Config<T>, Storage},
 		MmrLeaf: mmr_common::{Pallet, Storage},
+		BridgeRococoGrandpa: pallet_bridge_grandpa::{Pallet, Call, Storage, Config<T>} = 40,
+		BridgeWococoGrandpa: pallet_bridge_grandpa::<Instance1>::{Pallet, Call, Storage, Config<T>} = 41,
```

Comment:
I think an explanatory comment would be nice here, cause we are adding `Rococo` bridge pallet to `Rococo` runtime. It's only because Rococo and Wococo have exactly the same runtime, but the intention is to use `BridgeWococoGrandpa` pallet on Rococo and `BridgeRococoGrandpa` pallet on Wococo. The other pallet will always be uninitialized/halted.
This is a bit excessive and confusing, but I think it's acceptable for the matter of simplicity (no runtime duplication).